introduction.tex

```diff
@@ -62,6 +62,9 @@ \subsection{Register Definition Format}
         \hline
         R/W1 & Read/Write. Only writing 1 has an effect.  \\
         \hline
+        R/W1C & Read/Write. Writing every bit high clears the field. Writing
```

Comment:
Yeah, but I think the "don't unintentionally clear errors that you didn't actually observe" is the exactly desired behavior. 

Someone else suggested that you have to write exactly what is in the field in order to clear it.

Otherwise you have to go to 1-bit-per-error type, which is what I think you're suggesting.

Both of these are backwards incompatible.